<launch>

    <node pkg = "csrt_kf_new"   type = "new"    name = "Tracker"    output = "screen">

        <param name = "max_age"             value = "15"/>
        <param name = "min_hits"            value = "2"/>
        <param name = "view_img"            value = "false"/>
        <param name = "img_sub"             value = "/yolo/img"/>
        <param name = "bbx_sub"             value = "/yolo/bbx"/>

    </node>
</launch>
